docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+postgres:
+  image: sameersbn/postgresql
+  ports:
+  - "5432:5432"
+  environment:
```

Comment:
In docker-compose kan eenvoudig een `.env` file worden meegegeven: https://docs.docker.com/compose/environment-variables/